from dbt.artifacts.exceptions.schemas import IncompatibleSchemaError
